matsim/src/main/java/org/matsim/utils/objectattributes/attributeconverters/CoordConverter.java

```diff
@@ -44,7 +46,7 @@ public String convertToString(Object o) {
 		}
 		Coord c = (Coord)o;
 		
-		return String.format("(%.2f;%.2f)", c.getX(), c.getY());
+		return String.format(Locale.ROOT, "(%f;%f)", c.getX(), c.getY());
```

Comment:
I guess the `Locale` should be removed, because now you're writing "," instead "." as the decimal separator, which is dependent on you system settings.

Reply:
I'm not sure if I understand correctly but I want the formatted output to be independent of the user's system settings. So I should keep the `Locale.ROOT` right ?

Reply:
No, if you don't provide any Locale it will always use the standard English-style formatting.

Reply:
Mmm, that doesn't seem to be true... At least on my machine :
```
public class Test {
    public static void main(String []args){
        double toto = 13531.65165165;
        System.out.println(String.format("%f", toto));
    }
}
```
Returns : `13531,651652`

Reply:
Interesting, never noticed that. But than it would probably make sense to use US Locale so it is formatted with a dot, like in the code?

Reply:
(this is a small detail, but) I would argue It makes more sense to keep the root locale. According to the docs, it "is regarded as the base locale of all locales, and is used as the language/country neutral locale for the locale sensitive operations."
It does what we want (ie. put dots instead of commas in formatted double strings) and is country agnostic, which is relevant here.
Anyway thanks for the approval ! :)

Reply:
But when you used the root locale, you were testing for "1231,412" so it was actually not using a dot, didn't it? But I think now I see why I was confused. I think usually we always use `Double.parseDouble(String)` to read stuff in and then `Double.toString(Double)`, which then always does the same, independent of the system (for instance of all the `PopuatlionWriterV*` do it this way). So maybe for consistency's sake it would be best to say `String.format("%s;%s", Double.toString(x), Double.toString(y))` :)

Now we just need to figure out, why Travis doesn't want to run the PR ... Maybe you need to merge in the latest commits?

Reply:
Oooo `String.format("%s;%s", Double.toString(x), Double.toString(y))` is actually better as it keeps the double precision ! I will update and push with that method.